carbonmark/lib/constants.ts

```diff
@@ -22,21 +22,26 @@ export const WALLETCONNECT_PROJECT_ID =
   process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID;
 
 /** An abbreviated version of the commit hash used when deploying a preview build of the api in deploy_carbonmark_api.yml */
-const SHORT_COMMIT_HASH = process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA?.slice(
-  0,
-  7
-);
+// @todo uncomment before merge
+// const SHORT_COMMIT_HASH = process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA?.slice(
+//   0,
+//   7
+// );
 
 /** When incrementing this API version, be sure to update TypeScript types to reflect API changes */
-export const API_PROD_URL = "https://v5.0.0.api.carbonmark.com";
+export const API_PROD_URL = "https://v5.1.1.api.carbonmark.com";
```

Comment:
This does not match with the API version in the API PR

Reply:
I had initially understood the MO to be merge first and then open a separate PR that only increments the API version here? Open to whichever option, just want to confirm

Reply:
If we merge first then update staging will be broken in that intermediate time right?

Reply:
Just tried with the ICR FE pointing at the current api version and everything seemed(*) to work ok except everything ICR obviously.  Everything else being equal it makes sense to me(?) to just include the version incrementation in the APP PR itself.

And just to confirm the versioning:

Currently FE is on 5.0.0.

Currently API is 5.2.1

ICR API is 5.3.1

So the API PRs and APP PRs would increment both FE and API to 5.3.1. (seems like a big jump for the FE so just want to confirm).